repos/system_upgrade/common/actors/checkleftoverpackages/libraries/checkleftoverpackages.py

```diff
@@ -0,0 +1,43 @@
+import re
+
+from leapp.libraries.common.config.version import get_source_major_version
+from leapp.libraries.common.rpms import get_installed_rpms, get_leapp_dep_packages, get_leapp_packages
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledUnsignedRPM, LeftoverPackages, RPM
+
+
+def process():
+    LEAPP_PACKAGES = get_leapp_packages(major_version=[get_source_major_version()])
+    LEAPP_DEP_PACKAGES = get_leapp_dep_packages(major_version=[get_source_major_version()])
+    installed_rpms = get_installed_rpms()
+
+    if not installed_rpms:
+        return
+
+    to_remove = LeftoverPackages()
```

Comment:
I would rename the `to_remove` variable to e.g. `old_pkgs` or something like that. but it's a nitpick.

Reply:
I would maybe even take it a step further and name it something like `old_pkgs_to_remove` / `leftover_pkgs_to_remove`

Reply:
Also did this in other files.